v3/src/models/formula/formula-mathjs-scope.ts

```diff
@@ -119,7 +119,7 @@ export class FormulaMathJsScope {
   //     based on: // https://github.com/josdejong/mathjs/blob/develop/examples/advanced/custom_scope_objects.js ---
 
   get(key: string): FValue {
-    return this.extraScope?.has(key) ? this.extraScope.get(key) : this.dataStorage[key]
+    return this.extraScope?.get(key) ?? this.dataStorage[key]
```

Comment:
Is it possible the extraScope is used to override dataStorage with an undefined or null value for a key? Looking at the types of where it is used in `PlottedFunctionFormulaAdapter` that doesn't seem possible. I'm just checking since that would no longer be possible after this change.

Reply:
It's a good question, but `extraScope` is a `Map<string, FValue>`, where `FValue` is defined as `type FValue = string | number | boolean | Date | object` so unless our types are off, it can't be used to store `null` or `undefined`.